python/plugins/db_manager/db_manager_plugin.py

```diff
@@ -52,7 +52,7 @@ def initGui(self):
         else:
             self.iface.addPluginToMenu(QApplication.translate("DBManagerPlugin", "DB Manager"), self.action)
 
-        self.layerAction = QAction(QgsApplication.getThemeIcon('dbmanager.svg'), QApplication.translate("DBManagerPlugin", "Update Sql Layer"),
+        self.layerAction = QAction(QgsApplication.getThemeIcon('dbmanager.svg'), QApplication.translate("DBManagerPlugin", "Update Sql Layer…"),
```

Comment:
We should settle on either Sql or SQL through DB manager. 

You had another change with SQL being used: ` self.actionSqlWindow = self.menuDb.addAction(QIcon(":/db_manager/actions/sql_window"), self.tr("&SQL Window"),`. My vote goes to SQL :)